src/lib/parser.ts

```diff
@@ -83,7 +83,9 @@ export function parseOutput(gptString: string): ParsedOutput {
             !line.startsWith("//") &&
             !line.startsWith("#") &&
             line.trim().toLowerCase() !== "none" &&
-            !line.toLowerCase().includes("no command")
+            !line.toLowerCase().includes("no command") &&
+            [".", "?", "!"].includes(line[line.length - 1]) &&
+            !["(", ")", "_"].some((char) => line.includes(char))
```

Comment:
Is the assumption here that any tell user section won't include `()_` characters?

Reply:
yes, it's imperfect but reasonable approximation imo, both checks lean on the side of false negatives rather than false positive (favouring not displaying text to the user) which I think is the right way to do

Reply:
yes, it's imperfect but reasonable approximation imo, both checks lean on the side of false negatives rather than false positive (favouring not displaying text to the user) which I think is the right way to do